gh.go

```diff
@@ -53,6 +63,22 @@ func run(path string, env []string, args ...string) (stdOut, stdErr bytes.Buffer
 	return
 }
 
+func runInteractive(path string, env []string, args ...string) (err error) {
+	cmd := exec.Command(path, args...)
+	cmd.Stdin = os.Stdin
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	if env != nil {
+		cmd.Env = env
+	}
+	err = cmd.Run()
+	if err != nil {
+		err = fmt.Errorf("failed to run gh. error: %w", err)
```

Comment:
I don't mind that the original ExitError is wrapped here. The extra context makes it clear that it was the `gh` command that failed, and not code from the extension itself. Perhaps we can just make it a lighter wrapper:
```suggestion
		err = fmt.Errorf("gh: %w", err)
```

Reply:
I went for fmt.Errorf("gh execution failed: %w", err) for both Execs.